src/Microsoft.AspNetCore.SignalR.Core/IUserIdProvider.cs

```diff
@@ -3,8 +3,16 @@
 
 namespace Microsoft.AspNetCore.SignalR
 {
+    /// <summary>
+    /// A provider abstraction for getting the user ID from a <see cref="HubConnectionContext"/>.
```

Comment:
'A provider abstraction for configuring the "user ID" for a connection'

Then, some remarks on how this sets UserIdentifier, and that it the string returned here is the string expected by `IHubClients.User(...)`